binding.gyp

```diff
@@ -0,0 +1,29 @@
+{
+  "targets": [{
+    "target_name": "native-ledger-node",
+    "sources": [
+      "src/c/native_ledger_client.cc"
+    ],
+    "include_dirs": [
+      "<!(node -e \"require('nan')\")",
+      ".",
+      "bat-native-ledger/include/bat",
```

Comment:
I don't think you need this one. All headers should be `#include "bat/ledger/..."`

Reply:
cpplint was complaining that I should be using full header paths. For the comments below, I did have them the way you requested and cpplint prompted me to change them.